client/webserver/api.go

```diff
@@ -219,7 +219,13 @@ func (s *WebServer) apiLogin(w http.ResponseWriter, r *http.Request) {
 
 // apiLogout handles the 'logout' API request.
 func (s *WebServer) apiLogout(w http.ResponseWriter, r *http.Request) {
-	err := s.core.Logout()
+	err := s.core.RestartDEXConnections()
+	if err != nil {
+		s.writeAPIError(w, "logout error: %v", err)
+		return
+	}
+
+	err = s.core.Logout()
```

Comment:
I'm not quite following the reasoning here. Why are we reconnecting in order to sign out?

Reply:
Ah. Probably because it refreshes the order list. If so, this is pretty heavy handed. I think what we really need is a function `func (dc *dexConnection) hasActiveOrders() bool` that just says whether the `dexConnection` has any orders that are `< order.MakerRedeemed`.

Reply:
Sure, I'll look around with